scripts/db_setup.sql

```diff
@@ -23,4 +23,11 @@ CREATE TABLE IF NOT EXISTS messages (
   message varchar(10000) NOT NULL,
   message_timestamp timestamp(6) NOT NULL,
   sent_or_received TINYINT NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS friends (
+  username varchar(100) NOT NULL,
+  friend_display_name varchar(20) NOT NULL,

```

Comment:
These don't need to be pre-pended with friend right?

Reply:
Thats for clarity - username is the local user, friend_ is the friend's info. This will also be needed if we want to store friend's username.